packages/cli/src/databases/entities/AbstractEntity.ts

```diff
@@ -45,17 +40,10 @@ function mixinStringId<T extends Constructor<{}>>(base: T) {
 
 function mixinTimestamps<T extends Constructor<{}>>(base: T) {
 	class Derived extends base {
-		@CreateDateColumn(tsColumnOptions)
-		@IsOptional() // ignored by validation because set at DB level
-		@IsDate()
+		@Column(tsColumnOptions)
 		createdAt: Date;
 
-		@UpdateDateColumn({
-			...tsColumnOptions,
-			onUpdate: timestampSyntax,
-		})
-		@IsOptional() // ignored by validation because set at DB level
-		@IsDate()
```

Comment:
looks like `@BeforeUpdate` doesn't actually update timestamps for us, and we should revert to `@UpdateDateColumn`